third_party/boringssl/generate_build_files.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# Generate intermediate build files for Apple platforms.
+#
+# Usage:
+#
+#     third_party/boringssl/generate_build_files.sh
+#
+# Run from Themis source tree root.
+
+set -eu
+
+THIS_SCRIPT=third_party/boringssl/generate_build_files.sh
+
+die() {
+    for line in "$@"; do
+        echo 2>&1 $line
+    done
+    exit 1
+}
+
+# First of all, check that we're being launched from Themis source tree root.
+if [[ ! -f "$THIS_SCRIPT" ]]; then
+    die "$PWD is not Themis source tree root." \
+        "Please launch $THIS_SCRIPT from the root directory."
+fi
+
+# Make sure that all dependencies are in place. Give instructions in case
+# not everything is installed. See BoringSSL's "BUILDING.md" for details.
+if ! python2 --version >/dev/null 2>&1; then
```

Comment:
is it works with python3? ubuntu/debian don't install python2 as default anymore but still allow install it. but probably, it will not continue. So, did you try python3 with this script?

Reply:
The script *did not* work with Python 3 when I tested it. That's the reason for Python 2 usage.

However, now that you've reminded me about this PR, I have checked recent updates by Google. Lo and behold, [they seem to have updated their stuff](https://boringssl.googlesource.com/boringssl/+/666f2ab65e677bccc3db7063ae1f8706417341e1) to work with Python 3, just a couple of weeks ago. Some of it. So it might work with Python 3 now. Or might not. That needs testing.

Reply:
It seems to work with Python 3 now 🎉

```
divinity:themis ilammy$ make update_boringssl
Switched to branch 'master'
remote: Finding sources: 100% (3051/3051)
remote: Total 3051 (delta 1647), reused 3051 (delta 1647)
Receiving objects: 100% (3051/3051), 18.15 MiB | 3.65 MiB/s, done.
Resolving deltas: 100% (1647/1647), completed with 320 local objects.
From https://boringssl.googlesource.com/boringssl
   7be158d18..782d9b6cf  master            -> origin/master
[ ... ]
Updating 1607f54fe..782d9b6cf
Fast-forward
[ ... ]
 509 files changed, 42864 insertions(+), 42849 deletions(-)
[ ... ]
Generating build files...
Done!
Ignoring nokogiri-1.10.1 because its extensions are not built. Try: gem pristine nokogiri --version 1.10.1
Ignoring unf_ext-0.0.7.5 because its extensions are not built. Try: gem pristine unf_ext --version 0.0.7.5
divinity:themis ilammy$
```

Reply:
so can we switch to python3 instead of python2?